src/domain/creator/CreatorData.ts

```diff
@@ -0,0 +1,53 @@
+
+export default class CreatorData
+{
+    #id: string;
+
+    #fullName: string;
+    #nickName: string;
+    #email: string;
+    #portraitId: string;
+
+    #joinedAt: Date;
```

Comment:
As discussed, the purpose of the data model is for the initial design only. We've chosen to make the code the source of truth after the design phase to prevent additional overhead.